app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+syntax = "proto2";
+
+message SchedulerConfig {
+    // Targets to run from this config.
+    repeated Target targets = 1;
+    // Git branches to run these targets against.
+    repeated string enabled_branches = 2;
+}
+
+message Target {
+    // Unique, human readable identifier.
+    optional string name = 1;
```

Comment:
shall the name be required? is it builder name, task name, or any other?

Reply:
Protos should never use required field as it causes issues in the future if you deprecate the field ([go/protodosdonts](http://go/protodosdonts))